docs/fsharp/language-reference/type-extensions.md

```diff
@@ -115,7 +115,7 @@ There is no way to get this code to work with an optional type extension:
 
 * As is, the `Sum` member has a different constraint on `'T` (`static member get_Zero` and `static member (+)`) than what the type extension defines.
 * Modifying the type extension to have the same constraint as `Sum` will no longer match the defined constraint on `IEnumerable<'T>`.
-* Making changing the member to `member inline Sum` will give an error that type constraints are mismatched
+* Changing `member this.Sum`member to `member inline this.Sum` will give an error that type constraints are mismatched.
```

Comment:
There's still an unnecessary member here, @cartermp: Changing \`member this.sum\` to \`member inline this.Sum\` gives...

Reply:
Putting Ron's comment in suggestion form:
```suggestion
* Changing `member this.Sum` to `member inline this.Sum` will give an error that type constraints are mismatched.
```